src/Microsoft.IdentityModel.Tokens/ECDsaSecurityKey.cs

```diff
@@ -103,22 +103,7 @@ public override int KeySize
         /// <returns>true if this supports the algorithm; otherwise, false.</returns>
         public override bool IsSupportedAlgorithm(string algorithm)
```

Comment:
Should this be virtual? The method is more like `GetSignaureProvider` now, all it does is call into CPF. It will provide multiple ways of overriding the same thing.